db/schemas/manifestschema.json

```diff
@@ -0,0 +1,356 @@
+                    "$ref": "#/definitions/address" 
+                },
+                "has_discrepancy": {
+                    "description": "(18) Indication if there is a discrepency on the manifest",
+                    "type": "boolean"
+                },
+                "discrepancy": {
+                    "name": {
+                        "description":"(18b) Alternative designated facility company name as presented in RCRAInfo",
+                        "type": "string",
+                        "maxLength": 80
+                    },
+                    "us_epa_id_number":  {
+                        "description": "(18b) Unique tracking number assigned to each RCRA Handler/Site. Starts with 3 Letters and ends with 9 numbers",
+                        "type": "string",
+                        "maxLength": 12,
+                        "minLength": 12,
+                        "COMMENT": "add Regular Expressions"
+                    },
+                    "phone_number":  {
+                        "description": "(18b) Alternative facility contact phone number",
+                        "$ref": "#/definitions/telephonenumberdatatype" 
+                    },
+                    "category_quantity": {
+                        "description": "(18a) Indication if the discrepency is due to quantity",
+                        "type": "boolean"
+                    },
+                    "category_type": {
+                        "description": "(18a) Indication if the discrepency is due to type of waste recieved",
+                        "type": "boolean"
+                    },
+                    "category_residue": {
+                        "description": "(18a) Indication if the discrepency is due to residues",
+                        "type": "boolean"
+                    },
+                     "type":  {
+                        "description": "(18a) Type of rejection, no, partial, or full rejection",
+                        "type": "string",
+                        "enum":["no_rejection","partial_rejection","full_rejection"]
+                    },
+                        "description": {
+                        "description": "(14) Description of discrepancy",
+                        "type": "string"
+                    },
+                    "signatory":   {
+                        "description": "(18c) Alternate facility (or Generator) signature information",
+                        "$ref": "#/definitions/signature_info" 
+                    },
+                        "address": {
+                        "description": "(18b) Alternate facility address as presented in RCRAInfo",
+                        "$ref": "#/definitions/address" 
+                    }
+                    }
+                  },
+
+
+                "manifest_items": {
+                    "type": "array",
+                    "items": [{
```

Comment:
Looking at the JSON and form it is an array of one object that get's repeated.  manifest_items is a section of the form and all those pieces are part of that section.  A manifest can have one manifest item or many manifest items. On the form items 9a-13 (http://www3.epa.gov/epawaste/hazard/transportation/manifest/pdf/newform.pdf)